cluster.js

```diff
@@ -6,6 +6,7 @@ const log = require('./logger');
 
 const workers = [];
 let restart = true;
+var child;
```

Comment:
actually - this isn't used anywhere except the scope below. why not just use `let` below inside the loop?